example/vr.js

```diff
@@ -102,6 +102,19 @@ function init() {
 	tiles = new TilesRenderer( '../data/tileset.json' );
 	offsetParent.add( tiles.group );
 
+
+	// We define a custom scheduling callback to handle also active WebXR sessions
+	const tilesSchedulingCB = func => {
+
+		setTimeout( func, 50 );
+
+	};
```

Comment:
Thanks for updating the example! In order to best support the XR loop we'd have to do something like so, right?

```js
let _xrHandle = - 1;
let _rafHandle = - 1;
const tilesSchedulingCB = func => {

  if ( renderer.xr.isPresenting ) {

    const session = renderer.xr.getSession();
    session.cancelAnimationFrame( _xrHandle );
    _xrHandle = session.requestAnimationFrame( func );

  }

  cancelAnimationFrame( _rafHandle );
  _rafHandle = requestAnimationFrame( func );

};
```

I think this would be the most ergonomic way to support both rAF implementations. Thoughts @lojjic?

Reply:
Hmm it may be tricker than that; I think it may result in an orphaned rAF when the user switches from one mode to the other...? 🤔 

The `renderer.xr` looks like it fires 'sessionstart' and 'sessionend' events which you could possibly use to move any pending callbacks from one rAF over to the other.

Or maybe utilize `setAnimationLoop` in a queue/flush setup:

```js
let callbacks = [];

const tilesSchedulingCB = func => {
  callbacks.push(func);
};

threeRenderer.setAnimationLoop((...args) => {
  if (callbacks.length) {
    const funcs = callbacks;
    callbacks = [];
    funcs.forEach(cb => cb(...args));
  }
});
```

Reply:
Ok guys, so after the `setTimeout` setup for the VR example (that works fine), I also tested the **rAFs** approach, but without luck for active WebXR session - also it is a bit tricky like @lojjic said when switching between default and XR.

So I'm proposing something similar to the suggested callbacks (pending tasks consumed in the `setAnimationLoop`) for the VR example - that I think is simpler, and works fine for both standard and WebXR sessions (tested on different devices).

Furthermore if we find something even better in the future, it is an easy change in `tilesSchedulingCB`

Reply:
> I think it may result in an orphaned rAF when the user switches from one mode to the other...?

Thanks it occurred to me that this may be an issue a bit after I posted. Presumably there's a good reason why the WebXR rAF works independently from the global rAF this way otherwise this seems like a very awkward API quirk to work around.

The only other option that comes to mind is adding an "autoUpdate" option to the `PriorityQueue` that can be used to disable the auto scheduling requiring the user to trigger it themselves:

```js
tiles.downloadQueue.autoUpdate = false;
tiles.parseQueue.autoUpdate = false;

// ...

function animate() {

  tiles.downloadQueue.tryRunJobs();
  tiles.parseQueue.tryRunJobs();

  // ...

}
```

But I think we can switch to that another time if necessary. Unless you guys have other thoughts on the above I'll go ahead and test the PR over the next couple days and get this merged / published!